### DISCUSSION TITLE
Oxalis persistor plugin for Azure load error

### DISCUSSION THREAD

**Discussion by lskuba** [OxalisCommunity/oxalis#651] [General]
I am trying to add a plugin that will store messages and recipes in the Azure blob storage, I have tried to with my own implementation and also with one GitHub implementation:
https://github.com/data-altinn-no/oxalis-plugin-azureblob

I have made .jar file and placed into plugins ( jar appear in the docker under /oxalis/plugin/) : 
`COPY conf/oxalis-plugin-azureblob.jar /oxalis/plugin/`

Added two lines to the con: 
`oxalis.persister.receipt = plugin
oxalis.persister.payload = plugin`

But still I am getting error message:
`Exception in thread "main" com.google.inject.CreationException: Unable to create injector, see the following errors:

1) [Guice/ErrorInCustomProvider]: OxalisPluginException: Found 0 implementations of 'PayloadPersister'.
  at PersisterModule.getPluginPayloadPersister(PersisterModule.java:72)
  while locating PayloadPersister annotated with @Named(value="plugin")
  at PersisterModule.getPayloadPersister(PersisterModule.java:92)
  at DefaultPersisterHandler.<init>(DefaultPersisterHandler.java:56)
      \_ for 1st parameter
  at DefaultPersisterHandler.class(DefaultPersisterHandler.java:56)
  while locating DefaultPersisterHandler
  while locating PersisterHandler annotated with @Named(value="default")
  at PersisterModule.getPersisterHandler(PersisterModule.java:113)
  at As4FaultInHandler.<init>(As4FaultInHandler.java:38)`

I have also added  @Named(value="plugin") annotation to the (https://github.com/data-altinn-no/oxalis-plugin-azureblob/blob/master/src/main/java/no/nadobe/oxalis/plugin/azureblob/AzureBlobPersistor.java)  but still the same result. 

Any tip is welcome, thank you.

**Comment by aaron-kumar** [comment]
Not an issue.. Converting to discussion for further collaboration...
